pandas/io/formats/format.py

```diff
@@ -2115,6 +2118,15 @@ def _format_strings(self):
         return fmt_values
 
 
+class SparseArrayFormatter(GenericArrayFormatter):
+
+    def _format_strings(self):
```

Comment:
There was an issue with NumPy 1.10 and masking for sparse arrays. Seeing if this passes our CI quick, but I should be able to remove it.

edit: yep, everything passes. Let me see if I can simplify.